Assets/VRTK/Scripts/Interactions/VRTK_InteractTouch.cs

```diff
@@ -191,18 +195,22 @@ public Collider[] ControllerColliders()
             return (controllerCollisionDetector.GetComponents<Collider>().Length > 0 ? controllerCollisionDetector.GetComponents<Collider>() : controllerCollisionDetector.GetComponentsInChildren<Collider>());
         }
 
-        protected virtual void Awake()
-        {
-            controllerEvents = GetComponent<VRTK_ControllerEvents>();
-            controllerActions = GetComponent<VRTK_ControllerActions>();
-            VRTK_PlayerObject.SetPlayerObject(gameObject, VRTK_PlayerObject.ObjectTypes.Controller);
-            destroyColliderOnDisable = false;
-            var controllerHand = VRTK_DeviceFinder.GetControllerHand(gameObject);
-            defaultColliderPrefab = Resources.Load(VRTK_SDK_Bridge.GetControllerDefaultColliderPath(controllerHand));
+        protected virtual void Awake() {
+            if (this.isActiveAndEnabled && Application.isPlaying) Debug.Log("Touch Awake");
+            if (controllerEvents == null) controllerEvents = GetComponent<VRTK_ControllerEvents>();
+            if (controllerEvents == null) throw new System.Exception("InteractTouch requires non-null VRTK_ControllerEvents field controllerEvents.");
```

Comment:
The error message should probably be extended to explain that the field wasn't set to something (it's `null`) **and** there was nothing found on the current game object. Something along the lines of "InteractTouch requires a non-null `VRTK_ControllerEvents` instance. The field `controllerEvents` isn't set and there was no instance found on `gameObject`.